### PR TITLE
OCPBUGS-29777: RWOP accessMode is not available on OpenShift console UI

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
### Before:
Screenshot 2024-05-08 at 4 14 06 PM

### After:
Screenshot 2024-05-31 at 9 21 11 AM

Screenshot 2024-05-31 at 5 53 23 AM

### CODE REVIEW

**Comment by jhadvig** [comment]
@yuvalgalanti shall we keep using radio buttons here or lets switch to dropdown ?

File: frontend/public/components/storage/shared.ts [hunk 1]
```diff
@@ -23,31 +28,32 @@ type ProvisionerAccessModeMapping = {
   [provisioner: string]: AccessModeMapping;
 };
 
-// See https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes for more details
+// See https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes and
+// https://docs.openshift.com/container-platform/4.15/storage/understanding-persistent-storage.html for more details
 export const provisionerAccessModeMapping: ProvisionerAccessModeMapping = Object.freeze({
```
Comment (cyril-ui-developer) [inline frontend/public/components/storage/shared.ts:33]: FYI: Please verify PVC - Storage Class Provisioners below if any is missing the `ReadWriteOncePod` Access Mode?

**Review by bipuladh — APPROVED** [review]

**Comment by yuvalgalanti** [comment]
> @yuvalgalanti shall we keep using radio buttons here or let's switch to dropdown? 
I see what you are saying, since we have 4 options now I think having the access mode selection as a single select dropdown menu makes more sense and aligns with the convention in the OCP console.

**Review by vikram-raj — APPROVED** [review]

**Review by jhadvig — APPROVED** [review]
/lgtm

**Pull request merged**